dockers/docker-lldp/start.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 sonic-cfggen -d -t /usr/share/sonic/templates/lldpd.conf.j2 > /etc/lldpd.conf
+sonic-cfggen -d -y /etc/sonic/sonic_version.yml -t /usr/share/sonic/templates/lldpdSysDescr.conf.j2 >> /etc/lldpd.conf
```

Comment:
If @abdosi recommendation is not possible, please combine the calls to conic-cfggen into one call

Reply:
Thanks, I have combined the calls to sonic-cfggen into one call.

Reply:
@tahmed-dev : Please review the changes.